.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run a multi-line script
+      run: |
+        mvn -Pcodegen install
```

Comment:
Both maven commands can be merged to a single I think. Just merge all profiles `-Pcodegen,io.vertx,io.reactiverse` and the goals